data/mil2525d/core_data/gdbs/README.md

```diff
@@ -9,62 +9,64 @@ This folder contains the Military Features Core Geodatabase Schema format and an
-| credibility | J | J | Evaluation Rating | Credibility rating is second character of Evaluation Rating (J) field. |
-| datetimeexpired | W | W | Date-Time Group (DTG) | Second half of Date-Time Group (DTG) (W) field. |
-| datetimevalid | W | W | Date-Time Group (DTG) | First half of Date-Time Group (DTG) (W) field. |
-| distance | AM | | Distance | |
-| equipmentteardowntime | AE | | Equipment Teardown Time | |
-| higherformation | M | M | Higher Formation | |
-| idmode | P | P | IFF/SIF/AIS | |
-| platformtype | AD | AD | Platform Type | |
-| quantity | C | C | Quantity | |
-| reinforced | F | F | Reinforced or Reduced | |
-| reliability | J | J | Evaluation Rating | Reliability rating is first character of Evaluation Rating (J) field. |
-| sigintmobility | R2 | R2 | SIGINT Mobility Indicator | |
-| signatureequipment | L | L | Signature Equipment | |
-| speed | Z | Z | Speed | |
-| staffcomment | G | G | Staff Comments | |
-| type | V | V | Type | |
-| uniquedesignation | T | T | Unique Designation | |
-| uniquedesignation2 | T | | Unique Designation 2 | Used as an additional field for Fire Support Lines tactical graphics. |
-| x | Y | Y | Location | Longitude in degrees. |
-| y | Y | Y | Location | Latitude in degrees. |
-| z | X | X | Altitude/Depth | |
-| zmax | X | X | Altitude/Depth | Maximum altitude for aviation tactical graphics. |
-| zmin | X | X | Altitude/Depth | Minimum altitude for aviation tactical graphics. |
+| Attribute Name | 2525 D Field ID | Field Title | Notes |
+| ------------- | --- | ----------------- | ----- |
+| quantity | C | Quantity | |
+| reinforced | F | Reinforced or Reduced | |
+| staffcomment | G | Staff Comments | |
+| additionalinformation | H | Additional Information | |
+| additionalinformation2 | H1 | Additional Information 2 | Used in the Convoys and Airspace Coordination Area tactical graphics. |
+| credibility | J | Evaluation Rating | Credibility rating is second character of Evaluation Rating (J) field. |
+| reliability | J1 | Evaluation Rating | Reliability rating is first character of Evaluation Rating (J) field. |
+| combateffectiveness | K | Combat Effectiveness | |
+| signatureequipment | L | Signature Equipment | |
+| higherformation | M | Higher Formation | |
+| hostile | N | Hostile | |
+| idmode | P | IFF/SIF/AIS | |
+| direction | Q | Direction of Movement Indicator | |
+| sigintmobility | R2 | SIGINT Mobility Indicator | |
+| uniquedesignation | T | Unique Designation | |
+| uniquedesignation2 | T1 | Unique Designation 2 | Used as an additional field for Fire Support Lines tactical graphics. |
+| type | V | Type | |
+| datetimevalid | W | Date-Time Group (DTG) | First half of Date-Time Group (DTG) (W) field. |
+| datetimeexpired | W1 | Date-Time Group (DTG) | Second half of Date-Time Group (DTG) (W) field. |
+| z | X | Altitude/Depth | Maximum altitude for aviation tactical graphics. |
+| z1 | X1 | Altitude/Depth 2 | Minimum altitude for aviation tactical graphics. |
```

Comment:
WRT - lines 56-57 - concur with words for W and W1 for most symbol instances.  For CBRN release/attack, W = dtg start of release/attack and W1 = dtg for end of release/attack.  Many systems use W1 as a trigger to delete symbol from map; usually okay, bad for CBRN release/attack since the bad stuff is still present at that location